app/views/merchants/invoices/show.html.erb

```diff
@@ -15,6 +15,28 @@
       </tr>
   </table><br><br>
   <div>
-    <%= render partial: 'partials/invoice_items'%>
+    <h1> All Invoice Items</h1><br>
+    <h3>Expected Total Revenue: <%= number_to_currency(@invoice_items.total_revenue.to_f/100) %></h3> <br>
+    <table class="invoice-items"style='width: 80%; border-color: white;'>
+      <tr>
+        <th>Item</th>
+        <th>Quantity</th>
+        <th>Unit Price</th>
+        <th>Status</th>
+      </tr>
+        <% @invoice_items.each do |ii| %>
+          <tr>
+            <td><%= ii.item.name %></td>
+            <td><%= ii.quantity %></td>
+            <td><%= number_to_currency(ii.unit_price) %></td>
+            <td class="status-update"><%= form_with model: @invoice_item, method: :patch, local: true do |form| %>
```

Comment:
You can take out `method: patch`, it's implied through the context of the model.